src/toxcore/dht/packet/cookie.rs

```diff
@@ -235,18 +232,24 @@ mod tests {
             payload: invalid_payload_encoded
         };
         let decoded_payload = invalid_encrypted_cookie.get_payload(&symmetric_key);
-        assert!(decoded_payload.is_err());
-        assert_eq!(*decoded_payload.err().unwrap().kind(), GetPayloadErrorKind::Deserialize { error: ErrorKind::Eof, payload: invalid_encrypted_cookie.payload });
+        let error = decoded_payload.err().unwrap();
+        assert_eq!(*error.kind(), GetPayloadErrorKind::Deserialize {
+            error: Err::Error(("***************************************************".to_owned(), ErrorKind::Eof)),
```

Comment:
What do these stars mean?

Reply:
https://github.com/tox-rs/tox/pull/374/files#diff-2655c6396027ac3efcd54fe35a337272R32

https://github.com/tox-rs/tox/pull/374/files#diff-2655c6396027ac3efcd54fe35a337272R67

Reply:
Then it's wrong to convert it to string, it's an unparsed input which is not a string. It should be bytes array.